ingest_parsed_court_document_event_handler.tf

```diff
@@ -0,0 +1,47 @@
+locals {
+  ingest_parsed_court_document_event_handler_queue_name  = "${local.environment}-ingest-parsed-court-document-event-handler"
+  ingest_parsed_court_document_event_handler_lambda_name = "${local.environment}-ingest-parsed-court-document-event-handler"
+}
+module "ingest_parsed_court_document_event_handler_sqs" {
+  source     = "git::https://github.com/nationalarchives/da-terraform-modules//sqs"
+  queue_name = local.ingest_parsed_court_document_event_handler_queue_name
+  sqs_policy = templatefile("./templates/sqs/sqs_access_policy.json.tpl", {
+    account_id = var.dp_account_number, //TODO Restrict this to the SNS topic ARN when it's created
+    queue_name = local.ingest_parsed_court_document_event_handler_queue_name
+  })
+  redrive_maximum_receives = 3
```

Comment:
Standard SQS-Lambda complaint - max receives >= 5 and visibility timeout >= 6*lambda timeout.
...or use the Lambda Event Mapping scaling config.

Reply:
I remember thinking about this, then I forgot. I've changed this to 5. I've increased the lambda timeout to 30 seconds because 15 is cutting it close on a cold start and I've set the queue visibility timeout to 180.